lib/chainsync/chainsync/test_fixtures/db_session.py

```diff
@@ -1,20 +1,73 @@
 """Pytest fixture that creates an in memory db session and creates the base db schema"""
+import time
 from typing import Any, Generator
 
+import docker
 import pytest
-from chainsync.db.base import Base
-from sqlalchemy import create_engine
+from chainsync import PostgresConfig
+from chainsync.db.base import Base, initialize_engine
+from pytest_postgresql.janitor import DatabaseJanitor
 from sqlalchemy.orm import Session, sessionmaker
 
 
+@pytest.fixture(scope="session")
+def psql_docker() -> Generator[PostgresConfig, Any, Any]:
```

Comment:
what are the Any's here?

Reply:
Great question, I have no idea, this is the type returned from yields

Reply:
https://stackoverflow.com/questions/43658999/what-is-the-return-type-hint-of-a-generator-function

Will update to use `Iterator` here instead